### DISCUSSION TITLE
Configure tolerations for kured

### DISCUSSION THREAD

**Discussion by maaft** [mysticaltech/terraform-hcloud-kube-hetzner#1527] [Ideas]
### Description

Currently kured is not running on nodes with tolerations.

What is this repo using? The helm chart? Tolerations could be set there.

**Comment by mysticaltech** [comment]
@maaft We're using the manifest for now. But it would be better to use helm indeed. PR always welcome!

https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/0c87d28700a8e86f5012961284af6a0d5b29116f/locals.tf#L78

**Comment by maaft** [comment]
Thanks!

Shouldn't this stay an Issue though? Since currently there is no solution to configure tolerations for kured.
